src/main/java/com/amazon/dlic/auth/http/saml/SamlFilesystemMetadataResolver.java

```diff
@@ -51,6 +51,6 @@ public byte[] run() throws ResolverException {
 
     private static File getMetadataFile(String filePath, Settings settings, Path configPath) {
         Environment env = new Environment(settings, configPath);
-        return env.configFile().resolve(filePath).toAbsolutePath().toFile();
+        return env.configDir().resolve(filePath).toAbsolutePath().toFile();
```

Comment:
out of curiosity, did this get change in core or in one of the 3rd party dependencies?